src/sentry/utils/demo_mode.py

```diff
@@ -0,0 +1,49 @@
+from sentry import options
```

Comment:
two things:

- please don't add new things to `sentry.utils` -- it is a dumping ground
- please add new files to the typing stronglist